client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-detail-view/motion-detail-view.component.ts

```diff
@@ -221,6 +223,7 @@ export class MotionDetailViewComponent extends BaseMeetingComponent implements O
 
     public enterEditMotion(): void {
         this.editMotion = true;
+        this.originalMotion = JSON.parse(JSON.stringify(this.motion));
```

Comment:
This always seems hacky... What is the root problem? Is there another way to solve this?